packages/framer-motion/jest.setup.tsx

```diff
@@ -30,6 +30,10 @@ if (!(global as any).ResizeObserver) {
     } as any
 }
 
+// Stub scrollTo
+const noop = () => {}
+Object.defineProperty(global, "scrollTo", { value: noop, writable: true })
+
```

Comment:
Why do we need this in the PR?

Reply:
It's unrelated but removes some noise from the Jest console output